### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by chynu**
- Changed print logs to `logging.warn()`.
 - Removed `Actuator` and `Sensor` superclasses. Consolidated into `peripherals` directory.

### CODE REVIEW

**Comment by ScottSWu** [comment]
Run pylint

File: minibot/utils.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""
+Utils for minibot.
+"""
+
+class Utils:
```
Comment (ScottSWu) [inline minibot/utils.py]: Not really a fan of this file. `distance` is called only once, `normalize` is called twice.

File: minibot/peripherals/colorsensor.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        Gets name of sensor.
+        """
+        return self.name
+
+    def calibrate(self):
+        """
+        Calibrates colors. Takes 100 inputs per color, stores normalized average.
+        """
+
+        """ UPDATE: calibrate() no longer actually calibrates sensors. Method name is kept in case we
```
Comment (ScottSWu) [inline minibot/peripherals/colorsensor.py]: Remove all these comments

File: minibot/peripherals/colorsensor.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        #         print "Thank you! All of the colors have been calibrated.\n"
+
+        for c in self.colors:
+            self.colors[c] = Utils.normalize(self.colors[c])
+
+    def super_read(self, n):
+        """
+        Reads from the color sensor n times, and returns the non-normalized average.
+        """
+        color_data = {"R":0,"G":0,"B":0}
```
Comment (ScottSWu) [inline minibot/peripherals/colorsensor.py]: Using a dict here is excessive.
```
r = 0, g = 0, b = 0
for i in range(n):
  read = self.read()
  r += read[0]
  g += read[1]
  b += read[2]
...
```

File: minibot/peripherals/colorsensor.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        }
+        logging.warning("Sensor being registered: " + str(self.name))
+
+    def get_name(self):
+        """
+        Gets name of sensor.
+        """
+        return self.name
+
+    def calibrate(self):
```
Comment (ScottSWu) [inline minibot/peripherals/colorsensor.py]: This function is sorta useless

File: minibot/utils.py (hunk 1 above)
Comment (chynu) [reply minibot/utils.py]: Done.

File: minibot/peripherals/colorsensor.py (hunk 4 above)
Comment (chynu) [reply minibot/peripherals/colorsensor.py]: Removed.

File: minibot/peripherals/colorsensor.py (hunk 2 above)
Comment (chynu) [reply minibot/peripherals/colorsensor.py]: Removed function.

**Review by ScottSWu — APPROVED** [review]

File: minibot/peripherals/colorsensor.py (hunk 3 above)
Comment (chynu) [reply minibot/peripherals/colorsensor.py]: Lol tru. Done.

**Pull request merged**